scripts/docs/snippetadapter.js

```diff
@@ -261,6 +273,27 @@ function getWebpackConfig( snippets, config ) {
 		definitions[ definitionKey ] = JSON.stringify( config.definitions[ definitionKey ] );
 	}
 
+	const additionalLanguages = new Set();
+
+	// Find additional languages that must be built.
+	for ( const snippetData of snippets ) {
+		if ( snippetData.snippetConfig.additionalLanguages ) {
+			snippetData.snippetConfig.additionalLanguages.forEach( language => {
```

Comment:
I'd use `for ( const language of snippetData.snippetConfig.additionalLanguages || [] )` here to not mess `forEach` with `for of`.